presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -177,7 +177,7 @@ public StripeFooter readStripeFooter(List<OrcType> types, InputStream inputStrea
 
     private static Stream toStream(OrcProto.Stream stream)
     {
-        return new Stream(stream.getColumn(), toStreamKind(stream.getKind()), toIntExact(stream.getLength()), true);
+        return new Stream(stream.getColumn(), ColumnEncoding.DEFAULT_SEQUENCE_ID, toStreamKind(stream.getKind()), toIntExact(stream.getLength()), true);
```

Comment:
There should be no change in the reader. If the reader uses a method it is ok to leave them in the new Stream.